apps/cloud/src/context.rs

```diff
@@ -17,16 +20,23 @@ pub struct Context {
     pub storage: JwstStorage,
     pub user_channel: UserChannel,
     pub channel: BroadcastChannels,
-    _dir: Option<TempDir>,
+    _dir: Option<PathBuf>,
```

Comment:
we need keep the TempDir instance to delete the temp folder it before the program closes

Reply:
When using a temporary directory, expect the directory to be cleaned up after the program is closed

Reply:
I see, if DATABASE_DIR is set, set it to None.